package.json

```diff
@@ -157,9 +169,6 @@
     "prettier": "^1.19.1",
     "prop-types": "^15.7.2",
     "query-string": "^5.1.1",
-    "razzle": "3.1.1",
-    "razzle-config-utils": "^0.1.0",
-    "razzle-plugin-typescript": "^3.0.0",
```

Comment:
Disse trengs ikkje her fordi vi ikkje bruker razzle for kjøring, kun kompilering?

Reply:
Jeg antar det. Så på razzle eksempelet når jeg oppdaterte.

Reply:
https://github.com/jaredpalmer/razzle/blob/master/examples/with-typescript/package.json